NuRadioReco/modules/phasedarray/triggerSimulator.py

```diff
@@ -18,265 +19,284 @@
 default_sec_angles = np.insert(default_sec_angles, len(default_sec_angles), default_angles[-1] + 3.5 * units.deg)
 
 
-class triggerSimulator:
+def get_antenna_positions(station, det, triggered_channels=None, component=2):
```

Comment:
Needs docs (purpose, arguments, returns, etc.), and I think "component" as the argument is maybe not so clear? Also, I'm not sure why a function is needed just to return the depths of the antennas, so an explanation added would be good?